internal/controller/flow_falco.go

```diff
@@ -70,21 +70,21 @@ func parsePodNetworkInfo(input string) (*pb.FalcoFlow, error) {
 
 	layer3Message, err := createLayer3Message(info.SrcIP, info.DstIP, info.IpVersion)
 	if err != nil {
-		return nil, fmt.Errorf("unable to create Layer3 message falco flows: %v", err)
+		return &pb.FalcoFlow{}, err
```

Comment:
Why not just return `nil`?
```suggestion
		return nil, err
```
Same with the other error returns below.